tests/unit/test_zigzag_test_log.py

```diff
@@ -1555,6 +1632,17 @@ def test_attempt_parse_old_xml(self, test_without_test_step, mock_zigzag):
 class TestZigZagTestLogsTempest(object):
     """Tests for the TestLog class"""
 
+    def test_sample_tempest_xml_with_execution_parameters(self, tempest_case_xml_with_execution_parameters, mocker):
+        """Verify the status property when we expect a skip"""
+
+        zz = mocker.MagicMock()
```

Comment:
I'm seeing a mix of mock implementations for testing ZigZag test logs.  It definitely makes reading the tests even harder, is there some tech debt we should pay off in the future to make it more consistent? If so then let's ticket that tech debt.